modules/utxo-lib/src/bitgo/wallet/WalletOutput.ts

```diff
@@ -0,0 +1,74 @@
+import { taproot } from 'bitcoinjs-lib';
+import { UtxoPsbt } from '../UtxoPsbt';
+import { UtxoTransaction } from '../UtxoTransaction';
+import { RootWalletKeys } from './WalletKeys';
+import { ChainCode, scriptTypeForChain } from './chains';
+import { createOutputScript2of3, createPaymentP2tr, toXOnlyPublicKey } from '../outputScripts';
+
+/**
+ * Add a verifiable wallet output to the PSBT. The output and all data
+ * needed to verify it from public keys only are added to the PSBT.
+ * Typically these are change outputs
+ *
+ * @param psbt the PSBT to add change output to
+ * @param rootWalletKeys keys that will be able to spend the output
+ * @param chain chain code to use for deriving scripts (and to determine script
+ *              type) chain is an API parameter in the BitGo API, and may be
+ *              any valid ChainCode
```

Comment:
While we're moving code, we might as well get rid of this excess whitespace.

Reply:
Oh now with a slightly wider view I see that this whitespace is there to align the description of `chain` and not just some random whitespace. So we can leave as is.